our-brothers-web/src/app/components/meetings-list/meetings-list.component.html

```diff
@@ -30,14 +30,14 @@
       </div>
     </div>
     <div class="meeting-item-col meeting-participant">
-     כמות משתתפים: {{meeting.capacity}} 
+      {{meeting.capacity}} 
```

Comment:
It's best practices to not strings in CSS, because someday we will want to add localization, and it is done by HTML / JS files. Please divide it into 2 HTML and use the same class for the bereaved also. Something like that:
``` html
  <span class="mobile-label">
    כמות משתתפים: 
  </span>
```

``` css
.mobile-label {
  display: none;  

  @media only screen and (max-width: 600px) {
    display: inline;
  }
}
```